dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -937,6 +937,14 @@ STDMETHODIMP CFindFolder::MessageSFVCB(UINT uMsg, WPARAM wParam, LPARAM lParam)
             CComVariant searchBar(pwszGuid);
             return pWebBrowser2->ShowBrowserBar(&searchBar, NULL, NULL);
         }
+        case SFVM_WINDOWCLOSING:
+        {
+            if (m_shellBrowser)
+                m_shellBrowser = NULL;
```

Comment:
Is it possible to unconditionally do `m_shellBrowser = NULL;` (and same for the other pointer) without the if-tests?

Reply:
The documentation does not say either way but the code does check. I can take away the check here unless you say otherwise...

Reply:
Oh, it was just to simplify further the code, if possible.